extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -133,18 +133,24 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 	private readonly _requiredPackagesSet: Set<string>;
 
 	private readonly _runningOnSAW: boolean;
+	private readonly _tsgops_web: boolean;
```

Comment:
this variable isn't really following normal TypeScript naming conventions, which would be `_tsgopsWeb` or similar.